Documentation/cards/cards.html

```diff
@@ -117,7 +117,360 @@
     </div>
     </div>
     <div class="parent-container">
-        
+        <h2 class="font-xl">Cards</h2>
+
+        <h3 class="font-md">Text Card</h3>
+        <div class="component-display-div">
+          <div class="flex-row flex-center">
+          <div class="card p-1">
+            <div class="card-heading-container">
+            <h3 class="font-md font-medium">Chitale Ice Cream</h3>
```

Comment:
i think medium is for font-weight and md is for font-size, just a suggestion to have descriptive names

Reply:
will keep that in mind